app/models/contract.rb

```diff
@@ -1,5 +1,6 @@
 
 class Contract < ApplicationRecord
+  MAX_UNPAID_TIME = 6.months
```

Comment:
It would be nice to be configurable, with sane and safe defaults.

Reply:
Configurable why if we only use it for SaaS? 😕

Reply:
This is the OSS repository, SaaS/on-prem/OSS should have configuration to be used, it is a feature:
Delete accounts if they are inactive for X days ... blablabla

Exceptionally it is OK to have it fixed for 6.months but that should be some improvement later to let it be configurable

Reply:
We've had a conversation about this in Slack. Right now we are following the approach of not customising unless it is necessary, and Hery brought up that we intended to change the code to don't differentiate between SaaS and on-prem, and now we are Open Source, so our new approach should be allowing to customise always from now on.
Since this PR is pretty urgent and a bit more complicated that I'd like, and considering also that this would mean changing our approach from now on and not just for this time, I won't change it in this PR (unless someone disagrees and asks me to do it now) and right after we merge, I open a new PR for this 👍